app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/DateOfBirth.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models
+
+import java.time.LocalDate
+
+import play.api.libs.json.{Json, OFormat}
+
+final case class DateOfBirth(value: LocalDate)
```

Comment:
this isn't anything major, but you could do an optimisation here by extending `AnyVal` - this will make it so that at certain times instances of this case class aren't wrapped in a class. See here:

https://docs.scala-lang.org/overviews/core/value-classes.html

The same thing applies for things like implicit classes as well

Reply:
converted to a value class